apps/site/next-data/generators/__tests__/blogData.test.mjs

```diff
@@ -0,0 +1,189 @@
+import { Readable } from 'node:stream';
+
+import generateBlogData from '@/next-data/generators/blogData.mjs';
+
+let files = [];
+
+jest.mock('node:path', () => ({
+  ...jest.requireActual('node:path'),
+  join: jest.fn((_base, filePath) => filePath),
+}));
+
+jest.mock('node:fs', () => {
+  const originalFs = jest.requireActual('node:fs');
```

Comment:
We are only using one function from fs, so we don't need to add the original, right?

Reply:
yeah, I thought it would be cleaner to keep the originals too (to decrease the assumptions being made here)

but they are indeed not necessary, if you prefer I don't mind removing those :slightly_smiling_face::+1: 

done in https://github.com/nodejs/nodejs.org/pull/7618/commits/d62356553980802fee85ddc1cd7e82a20ad81746